tests/mechanic/telemetry_test.py

```diff
@@ -418,213 +418,324 @@ def test_stores_filtered_ccr_stats(self, metrics_store_put_doc):
+            "jvm_mem_heap_committed_in_bytes": 626393088,
+            "jvm_mem_heap_max_in_bytes": 626393088,
+            "jvm_mem_non_heap_used_in_bytes": 110250424,
+            "jvm_mem_non_heap_committed_in_bytes": 118108160,
+            "jvm_mem_pools_young_used_in_bytes": 66378576,
+            "jvm_mem_pools_young_max_in_bytes": 139591680,
+            "jvm_mem_pools_young_peak_used_in_bytes": 139591680,
+            "jvm_mem_pools_young_peak_max_in_bytes": 139591680,
+            "jvm_mem_pools_survivor_used_in_bytes": 358496,
+            "jvm_mem_pools_survivor_max_in_bytes": 17432576,
+            "jvm_mem_pools_survivor_peak_used_in_bytes": 17432576,
+            "jvm_mem_pools_survivor_peak_max_in_bytes": 17432576,
+            "jvm_mem_pools_old_used_in_bytes": 52336480,
+            "jvm_mem_pools_old_max_in_bytes": 469368832,
+            "jvm_mem_pools_old_peak_used_in_bytes": 52336480,
+            "jvm_mem_pools_old_peak_max_in_bytes": 469368832
+        }, flattened_fields)
+
+    @mock.patch("esrally.metrics.EsMetricsStore.put_doc")
+    def test_stores_default_nodes_stats(self, metrics_store_put_doc):
+        client = Client(nodes=SubClient(stats=NodeStatsRecorderTests.node_stats_response))
         cfg = create_config()
         metrics_store = metrics.EsMetricsStore(cfg)
         metrics_store_meta_data = {"cluster": "remote"}
         telemetry_params = {}
         recorder = telemetry.NodeStatsRecorder(telemetry_params, cluster_name="remote", client=client, metrics_store=metrics_store)
         recorder.record()
 
-        metrics_store_put_count.assert_has_calls([
-            mock.call(node_name="rally0", name="thread_pool_generic_threads", count=4, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="thread_pool_generic_queue", count=0, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="thread_pool_generic_active", count=0, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="thread_pool_generic_rejected", count=0, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="thread_pool_generic_largest", count=4, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="thread_pool_generic_completed", count=8, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="breaker_parent_overhead", count=1.0, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="breaker_parent_tripped", count=0, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="jvm_buffer_pool_mapped_count", count=7, meta_data=metrics_store_meta_data),
-            mock.call(node_name="rally0", name="jvm_buffer_pool_direct_count", count=6, meta_data=metrics_store_meta_data),
-        ], any_order=True)
+        # Extract mock call args, needed to perform dict comparison of first param of mocked method
```

Comment:
That's really too bad that we have to do this. Would it help if we use a sorted dict? If not, maybe define a helper method named e.g. `assertNodeStatsRecorded()` that encapsulates this (you also need this in the next test)?